app/javascript/components/mapping/MappingForm.jsx

```diff
@@ -346,6 +346,20 @@ const MappingForm = () => {
 
           <form onSubmit={handleSubmit}>
             <div className="form-group">
+              {files.length && !submitted && !processingFile && (
```

Comment:
Since the result of this expression gets rendered eventually, we need it to evaluate to a boolean value. Otherwise `0` will be displayed when there are no files. So use one of these:

- `!!files.length`,
- `files.length > 0`,
- `Boolean(files.length)`.